README.md

```diff
@@ -2,6 +2,9 @@
 [ ![Download](https://img.shields.io/badge/Download-iOS%20SDK-blue.svg) ](https://www.twilio.com/docs/video/ios#add-the-sdk)
 [![Docs](https://img.shields.io/badge/iOS%20Docs-OK-blue.svg)](https://twilio.github.io/twilio-video-ios/docs/latest/index.html)
 
+**NEW:** Please check out the newly open-sourced [multi-party video application](https://github.com/twilio/twilio-video-app-ios)
```

Comment:
I’m not sure we should call it `multi-party video application`. We already have the `MulitPartyExample` quick start application that we refer to in the docs. I am wondering if we should call it something else?

Reply:
@aaalaniz call it `video app`? I copied from JS quickstart.

Reply:
Or `Twilio Video iOS App`? That is what the repo title is.

Reply:
Let's go with `video collaboration app`